lib_web/static.ml

```diff
@@ -0,0 +1,21 @@
+let serve_file ~docroot uri =
+  let fname = Utils.Server.resolve_local_file ~docroot ~uri in
```

Comment:
I think it would be better to list the resources explicitly here. In theory, `resolve_local_file` will deal with `..` and similar, but I'm not sure relying on that is a good idea. Especially on Windows, where someone might ask to read `CON` or something.

Reply:
`resolve_local_file` does indeed handle `..` and similar (I wrote the unit tests for that recently).

The next line in that code is passing the resolved filename to `Utils.Server.respond_file`, which is `Cohttp_lwt_unix.Server.respond_file`.  That function checks that the given filename resolves to a real file (`Unix.st_kind = S_REG`) so it will not serve special files.

Also, this is not serving arbitrary files even from the specified docroot, because it is under the control of `Current_web.routes`, which restricts the URL to `/images/*`.  I think that is sufficiently constrained.